e2e/editor-tabs.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form'
+  );
+});
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
```

Comment:
```suggestion
```

We actually don't need this. Playwright will close the page automatically for us.